content/03-reference/01-tools-and-interfaces/01-prisma-schema/index.mdx

```diff
@@ -111,9 +111,10 @@ datasource db {
 }
 ```
 
+Variables can be expanded using the format specified by [dotenv-expand](https://github.com/motdotla/dotenv-expand).
```

Comment:
Maybe add an example in a code block here? The repo also makes you click through to https://github.com/motdotla/dotenv-expand/blob/master/test/.env so having a Prisma specific and relevant examples would improve this a lot.

Reply:
True, code added. Just in case, added a friendly tip (seo?) about paas or similar services.